package.json

```diff
@@ -27,8 +27,9 @@
     "version": "is-prerelease || npm run update-changelog && git add CHANGELOG.md",
     "watch": "npm-run-all -p watch:*",
     "watch:js": "npm run build:js -- -w",
-    "prepublish": "not-in-install && npm run build || in-install",
-    "precommit": "lint-staged"
+    "prepublish": "not-in-install && npm run build || in-install && npm-merge-driver install",
+    "precommit": "lint-staged",
+    "posttest": "shx cat test/dist/coverage/text.txt"
```

Comment:
I think files related to the build should live outside of `dist/` so they don't get published to npm. Perhaps `package.files` should only have `dist` as well?

Reply:
`test/dist` is where we currently build the test bundle, and we already gitignore it. I think it made logical sense to put coverage next to the test dist as well.

Reply:
Misread `test/dist` as `dist/`, ignore my comment